src/components/footnote/footnote.scss

```diff
@@ -0,0 +1,51 @@
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/theme';
+@use '../../mixins/focus';
+@use '../../mixins/ms';
+
+$footnote-offset: ms.step(-2);
+$footnote-offset-negative: calc(#{$footnote-offset} * -1);
+$footnote-offset-left-negative: calc(
+  #{$footnote-offset-negative} + #{ms.step(1)} * -1
+);
+
+/// Footnotes are list items with return links after their content
+///
+/// 1. Since footnotes are linked to using hash links, we give them a little
+///    vertical offset so the text isn't right at the top of the screen
+.c-footnote {
+  position: relative;
+  scroll-margin: calc(#{$footnote-offset} + #{ms.step(-2)}); /* 1 */
+
+  /// When a footnote is targeted by the URL hash we apply a background color to it
+  /// We use an absolutely positioned pseudo-element so our background will extend
+  /// beneath the list item ::marker
+  &:target::before {
+    background-color: color.$base-gray-lighter;
+    border-radius: size.$border-radius-medium;
+    bottom: $footnote-offset-negative;
+    content: '';
+    left: $footnote-offset-left-negative;
+    position: absolute;
+    right: $footnote-offset-negative;
+    top: $footnote-offset-negative;
+    z-index: -1;
+
+    @include theme.styles(dark) {
+      background-color: color.$brand-primary-dark;
+    }
+  }
+
+  /// We apply some basic styles to the return link.
+  /// These are so the links get nice focus styles.
+  .c-footnote-link-item__return-link {
```

Comment:
@Paul-Hebert Can you confirm there is no `:hover` state by design? I just noticed there wasn't a hover state, thanks! 😉

Reply:
Sure thing, I experimented with hover states but couldn't find anything that really worked well.

I tried using a scale transform but it got wobbly and awkward in some browsers. I thought about using a background color but that was a bit much.

In the end I figured it was fine since our regular links don't have hover effects. But... I'm open to ideas! I would like if there was a good hover effect to add.

Reply:
I see, thanks for the extra details. 

Hmm...I don't have a good idea to suggest. It's fine as-is. 👍